tmail-backend/jmap/extensions-cassandra/src/main/scala/com/linagora/tmail/james/jmap/label/CassandraLabelChangesConfiguration.scala

```diff
@@ -0,0 +1,26 @@
+package com.linagora.tmail.james.jmap.label
+
+import java.time.Duration
+import java.time.temporal.ChronoUnit
+
+import com.google.common.base.Preconditions
+import org.apache.commons.configuration2.Configuration
+import org.apache.james.util.DurationParser
+
+object CassandraLabelChangesConfiguration {
+  private val DEFAULT_TTL: Duration = Duration.ofDays(60)
+  val DEFAULT: CassandraLabelChangesConfiguration = CassandraLabelChangesConfiguration(DEFAULT_TTL)
+
+  def from(configuration: Configuration): CassandraLabelChangesConfiguration = {
+    val labelChangeTtl: Duration = Option(configuration.getString("label.change.ttl", null))
```

Comment:
Where's the doc for this conf?

Reply:
It seems TMail does not have a doc entry for additional configuration yet. I would add a docs for those.